### DISCUSSION TITLE
Mysql syntaxe error

### DISCUSSION THREAD

**Discussion by RTLERATIO** [syslog-ng/syslog-ng#4351] [Q&A] [answered]
Hello 
I have to upgrade syslog-ng 3.27 to 4.0.1 so I create and database mysql for the syslog-ng 4.0.1 and I get this message error when I try to restart syslog after typing : 
syslog-ng --syntax-only

Error parsing within destination, syntax error, unexpected LL_IDENTIFIER in /etc/syslog-ng/conf.d/basic.conf:27:5-27:9:
22        sql(
23          type("mysql")
24          host("localhost")
25          database("syslog")
26          table("logs")
27---->     user("root")
27---->     ^^^^
28          password("")
29          columns("timestamp", "facility", "severity", "message")
30        );
31      };
32

Included from /etc/syslog-ng/syslog-ng.conf:162:1-162:1:

I guess that an error of syntaxe using this patern can someone help me to fix that ?

**Comment by MrAnno** [comment, marked as answer]
The option you're looking for is called `username()`.
